scope.h

```diff
@@ -244,6 +244,9 @@ class Scope : public Mount
     bool IsAssumeOrthogonal() const;
     void HandleSanityCheckDialog();
     void SetCalibrationWarning(CalibrationIssueType etype, bool val);
+    bool ValidGuideRates(double RAGuideRate, double DecGuideRate);
+    bool m_CalDetailsValidated;
+    bool m_bogusGuideRatesFlagged;

```

Comment:
should we put these up above grouped with the rest of the member variables, like after line 150?

Reply:
Thanks for the quick review.  All these suggestions make sense, I'll go ahead and make the changes.